.travis.yml

```diff
@@ -57,8 +60,18 @@ matrix:
      python: 3.6
    - env: TOXENV=py36-dj110-wt110
      python: 3.6
+   - env: TOXENV=py36-dj110-wt111
+     python: 3.6
+   - env: TOXENV=py36-dj111-wt17
+     python: 3.6
+   - env: TOXENV=py36-dj111-wt18
```

Comment:
This version combination is not supported by Wagtail (https://github.com/wagtail/wagtail/blob/a9b7265784761d7bd9428b9721bfea9486370c30/docs/releases/upgrading.rst).